composer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "studiopress/social-profiles-widget",
+	"type": "wordpress-plugin",
+	"description": "Genesis Social Profiles Widget allows you to insert icon links to your social profiles in any widgetized area.",
+	"homepage": "https://github.com/studiopress/social-profiles-widget",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"php": "^5.3 || ^7",
+		"composer/installers": "^1"
+	},
+	"require-dev": {
+		"php": "^5.6 || ^7",
+        "dealerdirect/phpcodesniffer-composer-installer": "*",
```

Comment:
There seems to be a mix of tabs and spaces in a few lines in composer.json.

Reply:
Should be resolved in the last commit as well @dreamwhisper :)